consensus/raft/consensus.go

```diff
@@ -27,6 +27,8 @@ import (
 
 var logger = logging.Logger("consensus")
 
+var tracing bool
```

Comment:
Global variable is ugly. You can simply add an unexported field to the LogOp type and set it in baseLogOp. (just like the context).